CONTRIBUTING.md

```diff
@@ -13,24 +13,23 @@ milestone contains issues we think are interesting for you to work on. You can f
 [`help wanted`](https://github.com/FutureCitiesCatapult/TomboloDigitalConnector/issues?q=is%3Aopen+is%3Aissue+label%3A%22help+wanted%22) 
 tag. Feel free to check the other milestones in case you find something that excites you more.
 If none of the issues reflects your problem/feature create a new one following the 
-[issue template](https://github.com/FutureCitiesCatapult/TomboloDigitalConnector/blob/master/.github/ISSUE_TEMPLATE.md).
+[issue template](https://github.com/FutureCitiesCatapult/TomboloDigitalConnector/issues/new).
```

Comment:
Is the word `... template` need in the link name?

Reply:
Not really! a leftover 💃